lib/rest-adapter.js

```diff
@@ -341,7 +341,7 @@ RestAdapter.urlNotFoundHandler = function() {
   };
 };
 
-RestAdapter.errorHandler = function(options) {
+RestAdapter.strongErrorHandler = function(options) {
```

Comment:
We need to change the implementation to use the new strong-error-handler module under the hood, the method names (API) can stay the same, no need to change that.